mmv1/products/compute/api.yaml

```diff
@@ -727,6 +727,13 @@ objects:
                 name: 'headerName'
                 description: |
                   The header field name to match on when bypassing cache. Values are case-insensitive.
+      - !ruby/object:Api::Type::Enum
+          name: 'compressionMode'
+          description: |
+            The mode of compression to reduce the size of data sent from the Cloud CDN edge to the client.
```

Comment:
Where did this description come from? I see "Compress text responses using Brotli or gzip compression, based on the client's Accept-Encoding header." in the API.

Reply:
I didn't find the description in the [REST Reference Page](https://cloud.google.com/compute/docs/reference/rest/v1/backendBuckets) so I used the description of Dynamic Compression mentioned [here](https://cloud.google.com/cdn/docs/dynamic-compression#use-cases). Can you please share the link to the API doc you are referring to?

Reply:
https://cloud.google.com/compute/docs/reference/rest/beta/backendBuckets

Reply:
Thanks, I've updated the description and added a single test as suggested.